netcore/CoreFX.issues.rsp

```diff
@@ -676,18 +676,6 @@
 # https://github.com/mono/mono/issues/15153
 -nomethod System.Tests.ActivatorTests.TestingCreateInstanceObjectHandleFullSignature
 
-####################################################################
-##  System.Text.Json.Tests
-####################################################################
-
-# The JSON value is not in a supported DateTime format
--noclass System.Text.Json.Serialization.Tests.ObjectTests
--noclass System.Text.Json.Serialization.Tests.ArrayTests
--nomethod System.Text.Json.Serialization.Tests.DictionaryTests.DictionaryOfClasses
--nomethod System.Text.Json.Serialization.Tests.CacheTests.PropertyCacheWithMinInputsFirst
--nomethod System.Text.Json.Serialization.Tests.CacheTests.PropertyCacheWithMinInputsLast
--nomethod System.Text.Json.Serialization.Tests.ValueTests.ValueFail
```

Comment:
The `requires precise GC` group of tests can also be removed 🙂 (at the end of the rsp file)

Reply:
I remove the ones you fixed. I'm still planning to do more pruning and cleanup after successful test run.

Reply:
There are three tests left that are still missing the condition.